tests/factories/cht/generate.js

```diff
@@ -87,22 +116,24 @@ const createAdditionalPersons = (nbrPersons, clinic) => {
   return Array
     .from({ length: nbrPersons - 1 })
     .map((_, i) => {
-      const additionalPersonName = `${FIRST_NAMES[i % FIRST_NAMES.length]} ${LAST_NAMES[i % LAST_NAMES.length]}`;
+      const lastName = clinic.name.split(' ')[1];
```

Comment:
Instead of adding more calculations like this (composing data <---> decomposing). It's better to expose that last-name property in the clinic, so it's accessed like: `clinic.lastName`